core/names.js

```diff
@@ -114,8 +119,8 @@ Blockly.Names.prototype.populateVariables = function(workspace) {
  * Generate names for procedures.
  * @param {!Blockly.Workspace} workspace Workspace to generate procedures from.
  */
-Blockly.Names.prototype.populateProcedures = function(workspace) {
-  let procedures = Blockly.Procedures.allProcedures(workspace);
+Names.prototype.populateProcedures = function(workspace) {
+  let procedures = goog.module.get('Blockly.Procedures').allProcedures(workspace);
```

Comment:
Is this intentional? I'm just unfamiliar with this method of aquiring haha.

Reply:
Yeah. This is a way to get around circular dependencies. Requiring `Blockly.Procedures` directly causes a dependency cycle.